logstash-core-event-java/build.gradle

```diff
@@ -92,11 +99,17 @@ idea {
 }
 
 dependencies {
-    compile 'com.fasterxml.jackson.core:jackson-core:2.7.1'
-    compile 'com.fasterxml.jackson.core:jackson-databind:2.7.1-1'
+    compile 'com.fasterxml.jackson.core:jackson-core:2.7.3'
+    compile 'com.fasterxml.jackson.core:jackson-databind:2.7.3'
+    compile 'com.esotericsoftware:kryo:3.0.3'
+
     provided 'org.jruby:jruby-core:1.7.22'
     testCompile 'junit:junit:4.12'
     testCompile 'net.javacrumbs.json-unit:json-unit:1.9.0'
+
+    runtime 'com.fasterxml.jackson.core:jackson-databind:2.7.4'
+    runtime 'org.jruby:jruby-complete:1.7.22'
```

Comment:
we need to figure this whole jruby provided vs. runtime dependency thing.

I think it makes sense to have jmh profiling be its own submodule with its own dependencies. that way it can require what it needs and not interrupt java event's dependencies.

what do you think?